build.gradle.kts

```diff
@@ -218,14 +219,23 @@ tasks.getByName<Jar>("jar") {
     enabled = false
 }
 
-val supportedVersions = listOf("1.16.5", "1.17", "1.17.1", "1.18.2", "1.19", "1.19.1", "1.19.2", "1.19.3", "1.19.4", "1.20")
+val supportedVersions = listOf("1.16.5", "1.17.1", "1.18.2", "1.19.4", "1.20.1")
 tasks {
+    val lastSuccessfulBuildUrl = uri("https://ci.athion.net/job/FastAsyncWorldEdit/lastSuccessfulBuild/api/json").toURL()
+    val artifact = ((JsonSlurper().parse(lastSuccessfulBuildUrl) as Map<*, *>)["artifacts"] as List<*>)
+            .map { it as Map<*, *> }
+            .map { it["fileName"] as String }
+            .first { it.contains("Bukkit") }
+
     supportedVersions.forEach {
         register<RunServer>("runServer-$it") {
             minecraftVersion(it)
             pluginJars(*project(":plotsquared-bukkit").getTasksByName("shadowJar", false).map { (it as Jar).archiveFile }
                     .toTypedArray())
             jvmArgs("-DPaper.IgnoreJavaVersion=true", "-Dcom.mojang.eula.agree=true")
+            downloadPlugins {
```

Comment:
I assume old builds have to be deleted manually?

Reply:
How well does that work in case someone wants to e.g. use a custom FAWE build (when working on features that affect both plugins)?

Reply:
Id have said that this would be out of scope of the run-task but I can find a solution for that.

Reply:
I just wanted to know the new limitations, I guess it's fine to work around it in such case.

Reply:
I've created a PR at the repo for the run-task plugin to support Jenkins natively. Might be worth it to wait and see if it will be merged. If not, should be okay to merge this one as is.